exchanges/huobi/huobi_futures.go

```diff
@@ -1159,6 +1160,10 @@ func (h *HUOBI) FuturesAuthenticatedHTTPRequest(ep exchange.URL, method, endpoin
 		Verbose:       h.Verbose,
 		HTTPDebugging: h.HTTPDebugging,
 		HTTPRecording: h.HTTPRecording,
+	}
+
+	if err := h.SendPayload(ctx, request.Unset, func() (*request.Item, error) {
```

Comment:
This function also uses timestamp based signature/headers. I would advise inlining. Huobi has rate limits